src/seeThru.js

```diff
@@ -249,22 +249,25 @@
 		, requestAnimationFrame = window.requestAnimationFrame || getRequestAnimationFrame()
 		, cancelAnimationFrame = window.cancelAnimationFrame || getCancelAnimationFrame()
 		, drawFrame = function(recurse){
+			var image, alphaData, i, len, currentTime = video.currentTime;
+			
+			if (lastFrameTime !== currentTime) {
+				lastFrameTime = currentTime;

```

Comment:
Maybe a name like `lastDrawnFrame` would be more obvious to the reader. Apart from that I feel it'd be even clearer if we assign that value *post*-drawing (even if it makes no difference technically).